src/scss/_main.scss

```
@@ -3,6 +3,8 @@ body {
   justify-content: center;
   align-items: center;
   width: 100%;
+  background-image: url(../../public/background-image.png);
```

A spróbuj tak i zobacz czy działa 

```suggestion
  background-image: url(background-image.png);
```